space-cli/actions.go

```diff
@@ -26,43 +32,71 @@ func setLogLevel(loglevel string) {
 	}
 }
 
-func actionStartCode(c *cli.Context) error {
-	service, loginResp, err := cmd.CodeStart()
+func actionApply(c *cli.Context) error {
+	args := os.Args
+	if len(args) != 3 {
```

Comment:
I think you'll get just a single argument. Haven't you tested this?

Reply:
![image](https://user-images.githubusercontent.com/24411676/73807312-046afe80-47f2-11ea-8a56-57f41febd7a6.png)

Reply:
Cool